models/src/main/scala/coop/rchain/models/block/StateHash.scala

```diff
@@ -10,4 +11,12 @@ object StateHash {
   implicit class StateHashOps(bs: StateHash) {
     def base16String: String = Base16.encode(bs.toByteArray)
   }
+
+  implicit def ordering: Ordering[StateHash] =
+    Ordering.by((b: ByteString) => b.toByteArray.toIterable)
+
+  implicit val show = new Show[StateHash] {
+    def show(validator: StateHash): String =
+      Base16.encode(validator.toByteArray).take(10)
+  }
```

Comment:
But also all of these definitions (StateHashOps, ordering, show) can be removed. There is no sense to define them separately for aliases. Definitions for BlockHash alias works for both, so better to not use aliases in these situations.